source/overview/security.rst

```diff
@@ -1,37 +1,91 @@
+=====================================
 Security Overview
-------
+=====================================
+
+Mattermost offers modern communication behind your firewall to keep your data safe, secure, and private.
 
-Mattermost keeps your data safe, secure, and private with enterprise-class data security and control.
+Security Improvements 
+------
 
-Mattermost Enterprise Edition E20, E10, and Team Edition offer: 
+Mattermost is built on an open source platform whose security is under continual review by developers, IT administrators and security researchers accountable for deploying the software in their organizations. 
```

Comment:
`Mattermost is built on an open source platform whose security`  > `Mattermost is built on an open source platform, the security of which` ??